lib/src/widgets/movie_card.dart

```diff
@@ -55,3 +188,4 @@ class MovieCard extends StatelessWidget {
     );
   }
 }
+*/
```

Comment:
if you aren't using this commented code, please remove it

Reply:
Done